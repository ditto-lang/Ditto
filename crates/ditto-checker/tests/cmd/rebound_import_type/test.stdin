module Test exports (..)

import Foo (A)
import Bar (A)
