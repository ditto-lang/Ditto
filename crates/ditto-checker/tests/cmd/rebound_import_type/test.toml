bin.name = "ditto-checker-testbin-check-module"
args = ["rebound_import_type.ditto"]
fs.sandbox = true
status = "failed"
