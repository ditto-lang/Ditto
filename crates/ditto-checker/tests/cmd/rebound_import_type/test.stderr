
  × type `A` imported multiple times
   ╭─[rebound_import_type.ditto:2:1]
 2 │ 
 3 │ import Foo (A)
   ·             ┬
   ·             ╰── first imported here
 4 │ import Bar (A)
   ·             ┬
   ·             ╰── imported again here
   ╰────

